frontend/src/metabase/containers/NewItemMenu/NewItemMenu.unit.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import nock from "nock";
+import userEvent from "@testing-library/user-event";
+
+import { renderWithProviders, screen } from "__support__/ui";
+import { setupDatabasesEndpoints } from "__support__/server-mocks";
+
+import type { Database } from "metabase-types/api";
+import { createMockCard, createMockDatabase } from "metabase-types/api/mocks";
+
+import NewItemMenu from "./NewItemMenu";
+
+// eslint-disable-next-line react/display-name
+jest.mock("metabase/actions/containers/ActionCreator", () => () => (
```

Comment:
It might not hurt to use a named function here, so we don't need to disable the linter. Just saying.

```ts
() => function ActionCreator() { return <div data-testid="mock-action-editor" /> }